packages/scaffolding-cli/templates/src/csr/src/services/apiCaller.ts

```diff
@@ -19,6 +20,8 @@ export default function apiCaller<T>(
             return res.data.results
         })
         .catch(err => {
+            // appInsights: track error event
+            trackError(err)
```

Comment:
How do we split the events in app insights? is there custom telemetry params that we can send to determine the client?

Reply:
Custom telemetry meta data like project name and version are automatically logged on each tracking call. "@microsoft/applicationinsights-react-js" lib automatically understands React SPA pages.

Reply:
I will add the custom trace function now. Good shout.

Reply:
Added